py/bodynoline.py

```diff
@@ -15,5 +15,20 @@ def _bodynoline():
     Raises:
         TODO
     """
-
+    with table(cls="sk-table"):
```

Comment:
if we're not ready to commit this file yet, let's go ahead and leave changes on `bodynoline.py` out of the pull request.